v3/examples/server/main.go

```diff
@@ -274,6 +276,28 @@ func main() {
 	http.HandleFunc(newrelic.WrapHandleFunc(app, "/browser", browser))
 	http.HandleFunc(newrelic.WrapHandleFunc(app, "/async", async))
 	http.HandleFunc(newrelic.WrapHandleFunc(app, "/message", message))
+	http.HandleFunc("/log", func(w http.ResponseWriter, req *http.Request) {
+		app.RecordLog(newrelic.LogData{
```

Comment:
Could we make this cleaner by writing a function to do the work? I like the tidiness of the HandleFuncs being here without the actual logic muddying it up.